Server/Auth/ApiAuthorizationFilter.cs

```diff
@@ -25,15 +27,26 @@ public void OnAuthorization(AuthorizationFilterContext context)
 
             if (context.HttpContext.Request.Headers.TryGetValue("Authorization", out var result))
             {
-                var keyId = result.ToString().Split(":")[0]?.Trim();
-                var apiSecret = result.ToString().Split(":")[1]?.Trim();
+                var tokenType = result.ToString().Split(" ")[0].Trim();
 
-                if (DataService.ValidateApiKey(keyId, apiSecret, context.HttpContext.Request.Path, context.HttpContext.Connection.RemoteIpAddress.ToString()))
+                switch (tokenType)
                 {
-                    var orgID = DataService.GetApiKey(keyId)?.OrganizationID;
-                    context.HttpContext.Request.Headers["OrganizationID"] = orgID;
-                    return;
+                    case "Basic":
```

Comment:
What's the motivation for doing a switch with a single case here instead of a simple `if` conditional?

Reply:
I thought perhaps there would be support for Bearer vs Basic tokens at some point but I agree it’s premature optimization.

Reply:
That makes sense.